runtime/makelib/targets.mk.ftl

```diff
@@ -344,11 +344,15 @@ LIBCDEFS := $(word 1,$(wildcard $(foreach d,$(TPF_ROOT),$d/base/lib/libCDEFSFORA
 # We use sed in these scripts to retain only those lines that will be interesting
 # to omr/ddr/tools/getmacros; this reduces the total footprint of all *.i files by
 # over 1GB and reduces the workload of getmacros as it reads them.
+
+DDR_SED_COMMAND := \
+    sed -n -r -e '/^@?DDRFILE_BEGIN /,/^@?DDRFILE_END /{ /^@/!s/^/@/;p }'
```

Comment:
I think this can be simplified to
```
sed -n -e '/^DDRFILE_BEGIN /,/^DDRFILE_END /s/^/@/p' -e '/^@/p'
 ```
and after the OMR change lands, it can be simpler still
```
sed -n -e '/^DDRFILE_BEGIN /,/^DDRFILE_END /s/^/@/p'
```
Note we don't need to use the `-r` option.